### PR TITLE
Update readme for scala and python examples/imageclassification

### LINKED ISSUE #768: analytics-zoo/zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification：Readme need updates about "install from pip" / "download prebuilt package"
more details pls refer to #718

### PR DESCRIPTION
**Pull request by Deliadd**
fixed #768

### CODE REVIEW

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
 
-2. Prepare predict dataset
-
+## Prepare predict dataset
 Put your image data for prediction in one folder.
 
-3. Run the example
+## Run after pip install
+```bash
+export SPARK_DRIVER_MEMORY=10g
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: where's the command?

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
+```
+See [here](https://github.com/intel-analytics/analytics-zoo/tree/master/pyzoo/zoo/examples/textclassification#options) for more configurable options for this example.
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: wrong...

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 3]
```diff
@@ -36,3 +38,6 @@ spark-submit \
     --conf spark.executor.extraClassPath=${ZOO_JAR_PATH} \
     path/to/predict.py -f $imagePath --model $modelPath --topN 5 --partition_num ${partitionNum}
 ```
+See [here](https://github.com/intel-analytics/analytics-zoo/tree/master/pyzoo/zoo/examples/textclassification#options) for more configurable options for this example.
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: wrong...

File: pyzoo/zoo/examples/imageclassification/README.md (hunk 2 above)
Comment (hkvision) [reply pyzoo/zoo/examples/imageclassification/README.md]: just remove it

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 4]
```diff
@@ -36,3 +38,6 @@ spark-submit \
     --conf spark.executor.extraClassPath=${ZOO_JAR_PATH} \
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: need to remove these lines...

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 5]
```diff
@@ -1,34 +1,32 @@
...
---verbose \
---master $master \
---conf spark.executor.cores=1 \
---total-executor-cores 4 \
---driver-memory 200g \
---executor-memory 200g \
---class com.intel.analytics.zoo.examples.imageclassification.Predict \
-${ZOO_JAR_PATH} -f $imagePath --model $modelPath --partition 4 --topN 5
 ```
+See [here](https://github.com/intel-analytics/analytics-zoo/tree/master/zoo/src/main/scala/com/intel/analytics/zoo/examples/textclassification#options) for more configurable options for this example.
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: remove this...

File: pyzoo/zoo/examples/imageclassification/README.md (hunk 1 above)
Comment (hkvision) [reply pyzoo/zoo/examples/imageclassification/README.md]: 咦你居然还在线。。。

Comment (hkvision) [reply pyzoo/zoo/examples/imageclassification/README.md]: Yes. You need to add `python predict.py ....[fill with arguments]`

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 6]
```diff
@@ -1,38 +1,40 @@
...
+Run the following command for Spark local mode (MASTER=local[*]) or cluster mode:
+```bash
+export SPARK_HOME=the root directory of Spark
+export ANALYTICS_ZOO_HOME=the folder where you extract the downloaded Analytics Zoo zip package
+MASTER=...
+modelPath=... // model path
+imagePath=... // image path
+topN=... // top n prediction
+partitionNum=... // A suggestion value of the minimal partition number
+${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: should be `spark-submit-with-zoo.sh`

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 7]
```diff
@@ -1,34 +1,31 @@
...
 imagePath=... // image path
+${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
+     --master ${MASTER} \
+     --driver-memory 8g \
+     --executor-memory 8g \
+     --verbose \
+     --conf spark.executor.cores=1 \
+     --total-executor-cores 4 \           
+     --class com.intel.analytics.zoo.examples.imageclassification.Predict \
+     ${ZOO_JAR_PATH} -f $imagePath --model $modelPath --partition 4 --topN 5
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: do we need ${ZOO_JAR_PATH}?

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ modelPath=... // model path
 imagePath=... // image path
 topN=... // top n prediction
 partitionNum=... // A suggestion value of the minimal partition number
-${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
+${ANALYTICS_ZOO_HOME}/spark-submit-with-zoo.sh
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: opps.... we need /bin/...

Comment (hkvision) [reply pyzoo/zoo/examples/imageclassification/README.md]: also need \
just change the script name

Comment (Deliadd) [reply pyzoo/zoo/examples/imageclassification/README.md]: done

**Pull request merged**